src/Controller/Api/PersonController.php

```diff
@@ -494,6 +499,26 @@ public function getPerson(Person $person): Response
             'country' => $person->getCountry(),
             'organization' => $person->getOrganization(),
             'position' => $person->getPosition()
-        ]);
+        );
+
+        $account = $person->getAccount();
+        if ($account instanceof Account) {
+            $posix = $account->isPosix();
+            $posixUsername = $account->getUsername();
+        }
+
+        $isMe = (($currentAccount instanceof Account) && $person->getId() === $currentAccount->getPerson()->getId()) ? true : false;
+        $authenticatedData = array(
+            'isposix' => $posix,
+            'posixUsername' => $posixUsername,
+            'isMe' => $isMe
+        );
+
+        if ($currentAccount instanceof Account) {
+            $response = $this->makeJsonResponse(array_merge($anonymousData, $authenticatedData));
```

Comment:
Just merge the array here, but always make the response in the end.

Reply:
Also instead of merging, you can just add them to the array once you know it's the current user.